apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts

```diff
@@ -316,27 +316,37 @@ function RubricItemRow(item: RubricData['rubric_items'][0] | null, index: number
       </td>
       <td>
         <label
+          ${item ? html`for="rubric-item-explanation-button-${item.id}"` : ''}
```

Comment:
I'm a little wary about attaching a label to a button - is this valid? What does this achieve from an accessibility standpoint? And if it's for accessibility, why is it used conditionally?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label: 
> Elements that can be associated with a <label> element include <button>, <input> (except for type="hidden"), <meter>, <output>, <progress>, <select> and <textarea>.

It allows a user to click on the text and trigger the button action, i.e., editing of the content.

This is done conditionally because the content only exists if it is editing an existing rubric item, and I wanted to avoid an ID in the button (same conditional) inside the template used for new rubric items. I can also make the entire label conditional, if you'd prefer, in that case I would need to move the data attributes to a different object (e.g., the button).

Reply:
TIL! That usage makes sense.

I think I'd prefer the refactor you suggested, since I probably won't be the only person to ever look at that code and wonder about the conditional attributes.

Reply:
Done in 382ebfbb64b7d29407d83a05398dc1d970e6238b. I also took the opportunity to apply the auto-adjusting of height from the feedback into the explanation/grader note as well.